templates/cohorts/new_cohort_barcodes.html

```diff
@@ -122,7 +122,7 @@ <h5>Example Barcode File Contents</h5>
 TARGET-10-CAAABF-60.11C</pre>
                                     </ul>
                                     <a name="gdc-manifest"></a><h5>GDC Data Portal Case Manifest Files</h5>
-                                    <p>GDC Data Portal case manifests can be obtained on the <a target="_BLANK"
+                                    <p>GDC Data Portal case manifests can be obtained on the <a target="_blank" rel=“noreferrer”
                                                                                                 href="https://portal.gdc.cancer.gov/exploration">'Cases'
```

Comment:
So unfortunately this appears to have come through as smart-quotes and not straight quotes. D: I want to say it wouldn't matter but we should probably fix it. Most of the `rel=“noreferrer”` look this way.